### PR TITLE
[FLINK-22301][runtime] Statebackend and CheckpointStorage type is not shown in the Web UI

### PR DESCRIPTION
**Pull request by SteNicholas**
## What is the purpose of the change

*Currently we have already have the Rest API /jobs/:jobid/checkpoints/config that returns the statebackend name and checkpoint storage type:*
```
{
    "mode":"exactly_once",
    "interval":20,
    "timeout":600000,
    "min_pause":0,
    "max_concurrent":1,
    "externalization":{
        "enabled":false,
        "delete_on_cancellation":true
    },
    "state_backend":"EmbeddedRocksDBStateBackend",
    "checkpoint_storage":"FileSystemCheckpointStorage",
    "unaligned_checkpoints":false,
    "tolerable_failed_checkpoints":0
}
```
*But the two fields does not shown in the Web UI of checkpoint configuration. `job-checkpoints.component.html` should add the above mentioned fields for show.*

## Brief change log

  - *`job-checkpoints.component.html` adds two rows for the show of `state_backend` and `checkpoint_storage` fields.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (**not applicable** / docs / JavaDocs / not documented)

### CODE REVIEW

File: flink-runtime-web/web-dashboard/src/app/interfaces/job-checkpoint.ts [hunk 1]
```diff
@@ -115,6 +115,8 @@ export interface CheckPointConfigInterface {
     enabled: boolean;
     delete_on_cancellation: boolean;
   };
+  state_backend: any;
+  checkpoint_storage: any;
```
Comment (gaoyunhaii) [inline flink-runtime-web/web-dashboard/src/app/interfaces/job-checkpoint.ts:119]: Is it better to be declared as`string` ?

**Comment by gaoyunhaii** [comment]
Very thanks @SteNicholas  for opening the PR! I only have a minor issue. And have you tested it manually via starting a new cluster and check the UI ?

**Comment by SteNicholas** [comment]
@gaoyunhaii Yes, I have manually tests the changes for the statebackend name and checkpoint storage type.

**Review by tillrohrmann — APPROVED** [review]
Thanks for creating this PR @SteNicholas. LGTM. I've tested your change it worked for me. Merging this PR now.

**Pull request closed without merging**